src/input.rs

```diff
@@ -206,15 +207,24 @@ impl Action {
             },
             Action::Command(ref program, ref args) => {
                 trace!("running command: {} {:?}", program, args);
-                match Command::new(program)
+
+                #[cfg(not(windows))]
+                let spawned = Command::new(program)
                     .args(args)
                     .before_exec(|| {
                         // Detach forked process from Alacritty. This will cause
                         // init or whatever to clean up child processes for us.
                         unsafe { ::libc::daemon(1, 0); }
                         Ok(())
                     })
-                    .spawn()
+                    .spawn();
+
+                #[cfg(windows)]
+                let spawned = Command::new(program)
+                    .args(args)
+                    .spawn();
```

Comment:
Instead of having different implementations, maybe the better choice is to properly `wait` for the process exit? That should work on both unix and windows.

Reply:
I'm not actually sure of the purpose of having a command bound to a key, and haven't tested it myself.